docs/extensibility/form-section.md

```diff
@@ -103,6 +104,35 @@ In the example, the visibility for item price and color are analogous - the form
 
 Form widgets are used in the resource forms.
 
+### Alert
+
+Alert widgets display information for user using predefined types.
```

Comment:
```suggestion
Alert widgets display values using predefined types.
```
Can we use the same definition as for the display section? Just to stay consistent.